wordpress/wp-content/plugins/cds-base/classes/Modules/Site/SiteSettings.php

```diff
@@ -101,72 +96,124 @@ public function collectionSettingsPageInit()
             'collection-settings-admin' // page
         );
 
-        register_setting(
-            'site_settings_group', // option_group
-            'collection_mode',
-        );
-
         register_setting(
             'site_settings_group', // option_group
             'collection_mode_maintenance_page',
+            function ($input) {
+                if (is_int($input)) {
```

Comment:
Thinking we should use `return intval($input);` here --- tested using the existing and it doesn't hold the right value